gaphor/SysML/propertypages.py

```diff
@@ -28,53 +29,63 @@
 new_builder = new_resource_builder("gaphor.SysML")
 
 
-@PropertyPages.register(sysml.Requirement)
+@PropertyPages.register(RequirementItem)
```

Comment:
It's better to have 2 separate property pages: 1 for the `sysml.Requirement` and one for `RequirementItem`. When you select an item in the model browser, it will only show the property pages related to the model element. When you select the diagram item, it will show the "presentation" property page too.

If you set the order to a bit bigger value than the model element property page, it will show right below.

This is one of the (big) changes we did for the next release.

Reply:
Great, thanks so much for this suggestion; that makes perfect sense! I've made the requested change